muxinc/mux-stats-sdk-avplayer
# Builds MUXSDKStats as an XCFramework

## Zube
https://zube.io/muxinc/mux/c/11914
https://zube.io/muxinc/mux/c/13151

## Overview

* Builds the MUXSDKStats AVPlayer SDK as a XCFramework
* The product name for `MUXSDKStatsTv` is now also  `MUXSDKStats`
* Supports Catalyst builds. 

## TODO

- [x] Update example demo apps

## QA

- [x] Can use `MUXSDKStats.xcframework` in a Cocoapods/Obj-c project.
- [x] Can use `MUXSDKStats.xcframework` in a Cocoapods/Swift project.
- [x] Can use `MUXSDKStats.xcframework` in a Obj-c regular xcode project.
- [x] Can use `MUXSDKStats.xcframework` in a Swift regular xcode project.

Then

For *each* of the above, we need to test 
- [x] tvOS Simulator
- [x] iPhone hardware
- [x] tvOS hardware

Then ideally for each of these tests we would also make sure we can use the debugger (this will indicate that the dSYM files are being included properly in the projects but also that they are able to be found by the projects integrating our SDK.
